training/metax/llama3_8B-megatron/config/config_C500x1x8.py

```diff
@@ -0,0 +1,7 @@
+tokenizer_path = "llama3_8b_hf"
+localbs = 1
+train_steps = 300
+theoryflops = 312000000000000.0
```

Comment:
this is nvidia A100's TFLOPS, should be written with metax's TFLOPS, or run FlagPerf with correct figure and commit with "-1.0"